presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -46,7 +46,9 @@ Analysis and confirmation
 Dynamic filtering depends on a number of factors:
 
 * Planner support for dynamic filtering for a given join operation in Presto.
-  Currently inner and right joins with comparison join conditions and semi-joins with IN conditions are supported.
+  Currently inner and right joins with ``=``, ``<``, ``<=``, ``>``, ``>=`` or
+  ``IS NOT DISTINCT FROM`` join conditions and semi-joins
+  with IN conditions are supported.
```

Comment:
inline with previous line

Reply:
There are doc guidelines to stay within 80 chars, it doesn't affect the rendering of the line in the documentation.